pkg/instance/start.go

```diff
@@ -330,6 +353,12 @@ func watchHostAgentEvents(ctx context.Context, inst *store.Instance, haStdoutPat
 				err = xerr
 				return true
 			}
+
+			if showProgress && !cloudInitCompleted {
```

Comment:
Received `ev.Status.Running=true` event may be ignored here.
It should add checking `receivedRunningEvent && cloudInitCompleted` like following:
```golang
} else if ev.Status.Running {
    receivedRunningEvent = true
} else if receivedRunningEvent && cloudInitCompleted {
    ...
    return true
}
```